.pylintrc

```diff
@@ -3,13 +3,15 @@
 # A comma-separated list of package or module names from where C extensions may
 # be loaded. Extensions are loading into the active Python interpreter and may
 # run arbitrary code.
-extension-pkg-allow-list=cv2,numpy
+extension-pkg-allow-list=cv2,
```

Comment:
I'm probably wrong about this, but does the Shapely module that Chris used for the Off-Axis ODLC need to be added into this list for his code to pass?